docs/source/science/HLS/HLSL30.ipynb

```diff
@@ -2,23 +2,87 @@
  "cells": [
```

Comment:
<p>FYI on binder :</p><p> </p><ul><li>takes a long time to launch </li><li>it doesn't run -- I encountered a first error importing the packages :</li></ul><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[4], line 1
----&gt; 1 from maap.maap import MAAP
      2 maap = MAAP(maap_host='api.maap-project.org')
      3 import geopandas as gpd

ModuleNotFoundError: No module named 'maap'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Hmmm. Since it's running on mybinder instead of in the ADE, would it not have the MAAP packages then? So I guess if we make it accessible through another link, we'd have to have it install those packages?</p>

Reply:
Could maybe be removed entirely? Or I wonder why this specific tutorial has it

Reply:
<p>We could. The VEDA guidelines has a section like this where they access the notebook so we can add it to any notebook for ours. Didn't even think about mybinder not having the same MAAP stuff though, so it may need to be removed. Either that or maybe include a section on how to access the ADE and get to the tutorial from there? Not sure if that's an option either though since I'm not sure who all gets access</p>